### DISCUSSION TITLE
What does “Make Ship Happen” mean?

### DISCUSSION THREAD

**Discussion by Hector-Zhuang** [vercel/turborepo#2632] [Help] [answered]
![image](https://user-images.githubusercontent.com/12469549/200474502-0a3799a4-934b-45e3-8889-dcfcaac3dd9e.png)

I'm sorry for this is not a technical question.
As I am not a native speaker of English, I'm very curious about what does this headline mean in the index page.

**Comment by nathanhammond** [comment, marked as answer]
It's a pun in English. Shipping code (releasing code) is good, and you want it to happen fast and often.

**Reply by Hector-Zhuang** [reply]
Got it! Thx.
Actually, I'm trying to translate the docs to Chinese for my fellows. And here is another and the last question. 
There is a sentence in Quickstart: 
> On **large applications** Turbopack updates 10x faster the an Vite and 700x faster than Webpack. For **the biggest applications** the difference grows even more stark with updates up to 20x faster than Vite.

What do the `large applications` and `the biggest applications` refer to? `The biggest applications` is especially confusing. Does it mean websites like Hulu, Notion, and Target in the showcase?

**Reply by nathanhammond** [reply]
我識睇中文字所以如果你想我睇下話畀我知(zh-hk, 口語)

Large applications = 比較多components嘅網站 (30,000 components)
Biggest applications = 多過50,000 components

Make Ship Happen = 講廣東話我會用“事”（“屎”）翻譯

**Reply by Hector-Zhuang** [reply]
多謝，早知你識睇中文字，我就唔寫英文啦。我將 docs 翻譯如下：

Large applications = 體積較大的應用
Biggest applications = 體積巨大的應用
Make Ship Happen = 快碼揚鞭

「體積較大的應用」和「體積巨大的應用」肯定係有問題嘅，但我依家知系組件數了...
